apps/Codex/install

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+DIRECTORY="$(dirname "$(dirname "$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )")")"
+
+function error {
+  echo -e "\\e[91m$1\\e[39m"
+  exit 1
+}
+
+# Install Node.js first
+"${DIRECTORY}/manage" install-if-not-installed "Node.js" "$(dirname "$0")" || error "Failed to install Node.js"
+"${DIRECTORY}/pkg-install" "git" "$(dirname "$0")" || exit 1 
+
+
+cd $HOME
+# Remove Coedx folder first
+rm -rf Codex || sudo rm -rf Codex
+# Clone Codex repo
+git clone https://github.com/jcv8000/Codex
+cd ./Codex
+CODEX_DIR=$(pwd)
+
+# One time use, since `source ~/.bashrc` not working
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
```

Comment:
There is no need for the nvm bash completion here.

Reply:
Tested before, the `npm install` command won't work without the bash completion.